app/src/main/java/com/theopensourcefamily/chessclock/ClockState.kt

```diff
@@ -0,0 +1,6 @@
+package com.theopensourcefamily.chessclock
+
+sealed class ClockState
+object Stopped : ClockState()
+object WhiteRunning : ClockState()
+object BlackRunning : ClockState()
```

Comment:
You can squash the two commits: 
`git reset --soft HEAD~1`
`git commit --amend --no-edit`
`git push --force`

Reply:
Done!!! :D